benchmarks/benchmarks-stack/benchmarks-stack/lib/benchmarks-stack.ts

```diff
@@ -50,13 +50,7 @@ export class BenchmarksStack extends cdk.Stack {
     });
 
     const assetBucket = s3.Bucket.fromBucketName(this, 'AssetBucket', init_instance_sh.s3BucketName)
-    const canaryBucketName = "s3canary-temp-bucket";
-
-    const tempBucket = new s3.Bucket(this, 'TempBucketForCanary', {
-      removalPolicy: cdk.RemovalPolicy.DESTROY,
-      bucketName: canaryBucketName,
-      autoDeleteObjects: true,
-    });
+    const canaryBucketName = "crt-s3canary-bucket-123124136734";
```

Comment:
add a comment here to make sure people knows where the bucket it and the lifetime for the bucket to make sure we are not keep useless data.

Reply:
is bucket configured to remove any objects older than a day or 2?